p2p/kademlia/dht.go

```diff
@@ -619,7 +628,8 @@ func (dht *DHT) listen() {
 			case messageTypeStore:
 				data := msg.Data.(*queryDataStore)
 				dht.addNode(newNode(msg.Sender))
-				key := dht.store.GetKey(data.Data)
+				//TODO: Get pastel generated key
+				key := []byte("cf23df2207d99a74fbe169e3eba035e633b65d94")
```

Comment:
Hah? why it is hardcoded here? This is wrong!
It has to be the real key associated with this data

in the original code key is sha1 of data so they regenerate it here:

`
data := msg.Data.(*queryDataStore)
dht.addNode(newNode(msg.Sender))
**key := dht.store.GetKey(data.Data)**
`

We use key provided from outside, so we need a way here to find it

Reply:
this is resolved due to 

@a-ok123 
> Actually - my fault- the request to change the key in kademlia from internal sha1 to externally provided was due to the previous raptorq library we tried to use - it was generate
Ing very specific symbols (chunks) ids with some encoding info integrated into these ids
10:49
But since that library did not work- we switched to different library- the rust one that will be implemented as standalone service. This library- doesn’t even create symbols ids - it instead incorporates encoding parameters into symbols itself
10:49
So - we can create symbols ids any way we choose
10:50
As sha1 is so 2005, let’s do sha3 ;)